src/blocking.rs

```diff
@@ -306,6 +306,19 @@ impl BlockingClient {
         };
         Ok(self.agent.get(&url).call()?.into_json()?)
     }
+
+    /// Gets some recent blocks starting at the tip or at `height` if provided.
+    ///
+    /// The number of blocks depends on the backend itself but it should be at least 10.
```

Comment:
IIUC, the call usually returns 10 entries, but we can't guarantee that, e.g., for a `start_height` < 10. So, if there indeed other backends that default to return a number other than 10, maybe just leave this line out?

Reply:
`mempool.space` returns 15. I could always truncate to 10 if that makes things better maybe? For now I've just expanded the comment to explain that mempool returns a different value.